docs/contributor/pocs/ginkgo-to-go-testing/ginkgo-to-go-testing.md

```diff
@@ -0,0 +1,19 @@
+# Migrate Ginkgo tests to Go testing PoC
+
+This proof of concept demonstrates how to migrate Ginkgo tests to Go's built-in testing framework. The goal is to convert Ginkgo-style tests into the plain Go testing style while maintaining the same functionality and structure.
+
+## Built-in testing E2E test example
+
+[Service Name Enrichment](./service_name_test.go)
+
+## Gomega
+
+Gomega is a matcher library for Go that provides expressive assertions. Even though it is often used together with Ginkgo, it can be also used with the built-in `testing` package to write tests in a more readable way. It means that we can keep our Gomega matchers while migrating from Ginkgo to Go's testing framework. Note that in every test func we must call `gomega.RegisterTestingT(t)` to register the testing.T instance with Gomega.
+
+## Labels
+
+Labels are used to categorize tests. In Ginkgo, labels are often used to group tests together. In the Go testing framework, we can use go test args to achieve a similar effect.
```

Comment:
The first and second sentence seem to say basically the same - why?

Reply:
You are right - it's not needed.